packages/core/src/exchanges/fetch.ts

```diff
@@ -118,7 +123,9 @@ const createFetchSource = (operation: Operation, shouldUseGet: boolean) => {
     let ended = false;
 
     Promise.resolve()
-      .then(() => (ended ? undefined : executeFetch(operation, fetchOptions)))
+      .then(() =>
+        ended ? undefined : executeFetch(dispatchDebug, operation, fetchOptions)
```

Comment:
this is a potentially undefined argument, let's pass it as the last one instead

Reply:
```suggestion
        ended ? undefined : executeFetch(operation, fetchOptions, dispatchDebug)
```